modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -246,6 +246,20 @@ func (fsrv *FileServer) Provision(ctx caddy.Context) error {
 		fsrv.precompressors[ae] = p
 	}
 
+	// check sort options
+	for idx, sortOption := range fsrv.SortOptions {
+		switch idx {
+		case 0:
+			if sortOption != sortByName && sortOption != sortByNameDirFirst && sortOption != sortBySize && sortOption != sortByTime {
+				return fmt.Errorf("invalid sort option %s", sortOption)
+			}
+		case 1:
+			if sortOption != sortOrderAsc && sortOption != sortOrderDesc {
+				return fmt.Errorf("invalid sort option %s", sortOption)
```

Comment:
Could we make these two error messages different? They should say that the first value should be a field to sort by (name, dir, size, etc) and the second error should say that it should be a direction (asc or desc).

Reply:
I'm sorry, I was not thinking things through...
Thanks for your patience.